sample-queries/sample-queries.json

```diff
@@ -1432,6 +1432,16 @@
         "docLink": "https://developer.microsoft.com/en-us/graph/docs/api-reference/beta/api/channel_get_messagereply",
         "tip": "This query requires a group id of the Team, channel id of the corresponding channel of that Team, message id of the message of which you need the reply and the id of the specific reply. To find the group id, channel id, message-id and reply-id you can run: 1) GET https://graph.microsoft.com/beta/me/joinedTeams 2) GET https://graph.microsoft.com/beta/groups/{group-id-for-teams}/channels 3) GET https://graph.microsoft.com/beta/teams/{group-id-for-teams}/channels/{channel-id}/messages 4) GET https://graph.microsoft.com/beta/teams/{group-id-for-teams}/channels/{channel-id}/messages/{message-id}/replies",
         "skipTest": false
+    },    
+    {
+        "id": "0d0cddaf-aae2-4541-8104-1964d22a8da4",
+        "category": "Microsoft Teams (beta)",
+        "method": "GET",
+        "humanName": "apps installed for user",
+        "requestUrl": "/beta/users/{my-id}/teamwork/installedApps?$expand=teamsApp",
```

Comment:
instead of using `/users/{my-id}`, couldn't we use `/me` instead? see screenshot for an instance of this working :) 

i'm pretty sure `{my-id}` didn't already exist as a sandbox id because we are able to use /me/ to get our own user instance through the API

![image](https://user-images.githubusercontent.com/47487758/125517428-0deec46a-f987-44af-8991-fa235981c21f.png)

Reply:
1697c1f1d137183125754925e740c898150383ae